extensions/amp-sidebar/0.1/amp-sidebar.js

```diff
@@ -99,7 +99,19 @@ export class AmpSidebar extends AMP.BaseElement {
         this.close_();
       }
     });
-    //TODO (skrish, #2712) Add history support on back button.
+
+    // Invisible close button at the end of sidebar for screen-readers.
+    const screenReaderCloseButton = this.document_.createElement('button');
+    // TODO(aghassemi, #4146) i18n
+    screenReaderCloseButton.textContent = 'Close the sidebar';
+    screenReaderCloseButton.classList.add('-amp-screen-reader');
+    // This is for screen-readers only, should not get a tab stop.
+    screenReaderCloseButton.tabIndex = -1;
```

Comment:
no, I don't want this to be used to hide hide things. There are cases where you want to apply ''-amp-screen-reader" to an existing visible node and then remove ''-amp-screen-reader" later to make the node visible again for everyone. Let's do the refactoring later when we have more uses of this, I think refactoring this right now might be premature.